tests/core/test_farmer_harvester_rpc.py

```diff
@@ -64,38 +61,6 @@ def wait() -> bool:
     await time_out_assert(30, wait)
 
 
-HarvesterFarmerEnvironment = Tuple[Service[Farmer], FarmerRpcClient, Service[Harvester], HarvesterRpcClient, BlockTools]
-
-
-@pytest_asyncio.fixture(scope="function")
-async def harvester_farmer_environment(
```

Comment:
did you move this verbatim, or did you make changes to it as well?

Reply:
Yes, this was moved verbatim.